.changeset/red-foxes-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'nextra-theme-blog': major
+---
+
+remove `nextra-theme-blog/cusdis` export, export `<Comments>` component from `nextra-theme-blog` directly
```

Comment:
```suggestion
remove `nextra-theme-blog/cusdis` export, export `<Comments>` component from `nextra-theme-blog` directly (because `peerDependency` of `react-cusdis` was removed)
```